packages/org.json_schema.contrib/internal/ModuleGenerator.pkl

```diff
@@ -240,7 +241,7 @@ local classSchemas: Type.TypeNames =
       .fold(Map(), (accumulator: Type.TypeNames, pair) ->
         let (path = pair.first)
         let (schema = pair.second)
-        let (typeName = determineTypeName(path, path.lastOrNull?.capitalize() ?? "Item", accumulator.values.toSet(), 0))
+        let (typeName = determineTypeName(path.dropLast(1), path.lastOrNull?.capitalize() ?? "Item", accumulator.values.toSet(), 0))
```

Comment:
Could you explain why this change is necessary?

Reply:
Otherwise, the generated classes end up with the property name repeated twice. You can check out the code snippet [here](https://github.com/apple/pkl-pantry/blob/be26d4171c97b5dca365a90a6397ff2739c5933c/packages/org.json_schema.contrib/internal/ModuleGenerator.pkl#L230C42-L230C52), where the first argument is appended with the current candidate name.

Reply:
Ah yes, I can see this in my output now too. Thanks for clarifying